meta-sokol-flex-support/recipes-core/images/flex-image.inc

```diff
@@ -13,6 +13,7 @@ LICENSE = "MIT"
 
 inherit core-image
 
-IMAGE_FEATURES:append:sokol-flex = " splash ${@bb.utils.contains('COMBINED_FEATURES', 'alsa', ' hwcodecs tools-audio', '', d)}"
+IMAGE_FEATURES:append:sokol-flex = " splash ${@bb.utils.contains('COMBINED_FEATURES', 'alsa', ' tools-audio', '', d)}"
+IMAGE_FEATURES:append:sokol-flex = " splash ${@bb.utils.contains('USER_FEATURES', 'multimedia', ' hwcodecs', '', d)}"
```

Comment:
multimedia is a distro feature, we should be checking `DISTRO_FEATURES` directly here.

Reply:
Isn't it an IMAGE_FEATURE https://github.com/MentorEmbedded/meta-sokol-flex/blob/main/conf/local.conf.sample#L43

Even if it were distro feature IIRC we've had some issues with checking USER_FEATURES added distro features inside the distro conf, hence this: https://github.com/MentorEmbedded/mel-meta-mentor-security/blame/fir_async2/meta-mentor-integrity/conf/distro/include/mel-security-ima-evm.conf#L22

Reply:
Ok @ahsanhussain, I have made the change as you recommended.

Appending an IMAGE_FEATURES to an IMAGE_FEATURE started giving me error:
RecursionError: maximum recursion depth exceeded while calling a Python object.

So I used IMAGE_INSTALL instead.